src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chromeablestage/tabgroups/ChapterTabGroup.kt

```diff
@@ -28,6 +28,11 @@ class ChapterTabGroup : TabGroup() {
         tabMap.clear()
     }
 
+    private fun shouldRestoreActiveResourceMetadata(metadataToRestore: ResourceMetadata?): Boolean {
+        return metadataToRestore != null
+            && getAssociatedMetadatas().toList().contains(metadataToRestore)
```

Comment:
Fails linter

Reply:
Can probably do this on the sequence without building a list with `toList` (but the method might not be `contains`).

Reply:
Actually `contains` is a method of a sequence- my code completion took too long to load and only loaded a few methods so I thought it didn't exist!